packages/datepickers/src/styled/utils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { css, DefaultTheme, ThemeProps } from 'styled-components';
+
+export const retrieveSpacing = ({
+  isSmall,
+  theme
+}: { isSmall?: boolean } & ThemeProps<DefaultTheme>) => {
+  if (isSmall) {
+    return `${theme.space.base * 8}px`;
+  }
+
+  return `${theme.space.base * 10}px`;
+};
```

Comment:
The fact that this value is subsequently being `math`-multiplied feels hacky and difficult to make sense of when reading through the CSS blocks where it is being called. The value of this utility is questionable – I almost wonder if you should provide a base `Styled` components that provide consistent height/width and can be extended.